nosepipe.py

```diff
@@ -147,12 +147,16 @@ def __call__(self, result):
             useshell = True
 
         self.logger.debug("Executing %s", " ".join(argv))
-        popen = subprocess.Popen(argv,
-                                 cwd=self._cwd,
-                                 stdout=subprocess.PIPE,
-                                 stderr=subprocess.STDOUT,
-                                 shell=useshell,
-                                 )
+        try:
+            popen = subprocess.Popen(argv,
+                                     cwd=self._cwd,
+                                     stdout=subprocess.PIPE,
+                                     stderr=subprocess.STDOUT,
+                                     shell=useshell,
+                                     )
+        except OSError, e:
```

Comment:
I should note, after installing this I just ran through my normal tests with nosetests to make sure nothing was broken with this change. I don't normally run via setup.py. Does this now fix your issue without problems? I'm confused by your last comment.